src/js/angular/clustermanagement/controllers/cluster-management.controller.js

```diff
@@ -108,6 +108,9 @@ function ClusterManagementCtrl($scope, $http, $q, toastr, $repositories, $uibMod
                 const nodes = cluster.getUpdateActions();
                 editCluster(nodes);
             }
+        }).catch((error) => {
+            $scope.setLoader(false);
```

Comment:
Well. This will do the job, just we usually set these flags down in finally blocks. Don't change it anymore here.